javascript/exosecurity/features/support/env.js

```diff
@@ -0,0 +1,30 @@
+const {
+  After,
+  Before,
+  setDefaultTimeout,
+  setWorldConstructor,
+} = require('cucumber')
+const { killChildProcesses } = require('exosphere-shared')
+const World = require('./world')
+
+setDefaultTimeout(1000)
+setWorldConstructor(World)
+
+function closeIfDefined(obj, done) {
+  if (obj) {
+    obj.close(done)
+  }
+  done()
```

Comment:
This looks wrong to me. I believe this needs an `else` to prevent a fall through.